apps/api/src/domain/models/song/commands/create-song.command-handler.ts

```diff
@@ -71,10 +70,11 @@ export class CreateSongCommandHandler extends BaseCommandHandler<Song> {
         return getInstanceFactoryForResource<Song>(ResourceType.song)(songDTO);
     }
 
-    actOnInstance(song: Song): ResultOrError<Song> {
-        return song;
-    }
-
+    /**
+     * TODO Optimize this by using filters to pull out only the necessary
```

Comment:
Seems like duplicate song titles should be allowed.  I think there are many Raven songs for example